contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -433,6 +434,7 @@ def publish(self, request, pk=None):
         channel = self.get_edit_object()
 
         if (
+            channel.main_tree.publishing or
```

Comment:
This was missing, and made sense to me: I don't think we want to allow multiple publish tasks to be created while publishing. I don't think that's possible from the frontend, but perhaps a good check nonetheless.